webknossos/webknossos/dataset/_utils/pims_images.py

```diff
@@ -522,22 +526,22 @@ def copy_to_view(
 
         with self._open_images() as images:
             if self._iter_axes is not None and self._iter_loop_size is not None:
-                # select the range of images that represents one xyz combination
+                # select the range of images that represents one xyz combination in the mag_view
                 lower_bounds = sum(
                     self._iter_loop_size[axis_name]
-                    * relative_bbox.get_bounds(axis_name)[0]
+                    * absolute_bbox.get_bounds(axis_name)[0]
```

Comment:
is this correct? I would have assumed that an absolute bbox is still required here (same as in lines 517ff). otherwise, `lower_bounds` would contain different values with this PR, even though its usage in line 536 is unchanged.

Reply:
Good catch, I think it didn't fail for the tested cases, as topleft was always 3D. So there was no offset for the additional axes. As soon as there are topleft offsets for additional axes, this will be a problem. I will fix it and add a test for it.

Reply:
You meant relative bbox, right? I changed it, so it should be correct now.

Reply:
great, thank you =)

> You meant relative bbox, right?

yes, my bad!